frontend/src/components/CreatePost/CreatePost.css

```diff
@@ -1,11 +1,9 @@
 .create-post {
   background-color: var(--white-shade-0);
-
   border: 1px solid var(--white-shade-2);
   width: 1000px;
```

Comment:
setting width in pixels will cause responsiveness issues, use % or vw instead in this case so it automatically scales for small screens